opensrp-giz-malawi/src/main/java/org/smartregister/giz/configuration/AllClientsRegisterQueryProvider.java

```diff
@@ -23,14 +27,15 @@
     public String getObjectIdsQuery(@Nullable String filters, @Nullable String mainCondition) {
         Date latestValidCheckInDate = OpdLibrary.getInstance().getLatestValidCheckInDate();
         String oneDayAgo = OpdUtils.convertDate(latestValidCheckInDate, OpdDbConstants.DATE_FORMAT);
+        String todayDate = new SimpleDateFormat(GizConstants.DateTimeFormat.yyyy_MM_dd, Locale.US).format(new Date());
 
         if (!TextUtils.isEmpty(filters)) {
             if (!TextUtils.isEmpty(mainCondition)) {
                 String sql =
                         "SELECT ec_client_search.object_id, ec_client_search.last_interacted_with, (opd_details.current_visit_start_date IS NOT NULL AND opd_details.current_visit_start_date >= '$latest_start_visit_date' AND opd_details.current_visit_end_date IS NULL) AS checked_in FROM ec_client_search WHERE date_removed IS NULL AND phrase MATCH '%s*') " +
-                        "    LEFT JOIN opd_details ON ec_client_search.object_id = opd_details.base_entity_id\n" +
-                        "WHERE checked_in " +
-                        "ORDER BY last_interacted_with DESC";
+                                "    LEFT JOIN opd_details ON ec_client_search.object_id = opd_details.base_entity_id\n" +
+                                "WHERE checked_in " +
```

Comment:
Are you still using `checked_in` for the new workflow?

Reply:
removed